tests/lib/rules/jsx-no-hardcoded-content.js

```diff
@@ -211,7 +211,6 @@ ruleTester.run('jsx-no-hardcoded-content', rule, {
     },
     {
       code: `
-        import {MyComponent} from 'my-module';
```

Comment:
Removed these because it was throwing an error `message: 'Parsing error: Identifier \'MyComponent\' has al
ready been declared`

Reply:
Makes sense 👍